composer.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "humanmade/hm-limit-login-attempts",
+    "description": "The original Limit Login Attempts plugin with the option to limit via username as well.",
+    "type": "package",
+    "license": "GPL-3.0-or-later",
+    "authors": [
+        {
+            "name": "Human Made",
+            "email": "hello@humanmade.com"
+        }
+    ],
+    "autoload": {
+        "files": [
```

Comment:
I'll update this but you have to be careful not to autoload files that have any side effects. In the case of the plugin file it calls `add_action()` without checking if the function exists so would break when running any other composer script or phpunit for example.